### PR TITLE
abrute man page

### PR DESCRIPTION
**Pull request by Alcamech**
Please provide feedback

### CODE REVIEW

File: abrute [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.\" Manpage for abrute
+.\" Contact alcamech@gmail.com to correct errors or typos.
+.TH man 1 "12 October 2017" "1.0" "abrute man page"
```
Comment (danielpclark) [inline abrute]: Change `man` to `abrute`, drop the "1.0", and change the last quoted part to just "Abrute"

File: abrute [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+.SH REPORTING BUGS
+Report bugs to <https://github.com/danielpclark/abrute/issues>
+.SH BUGS
```
Comment (danielpclark) [inline abrute]: At this time we don't need a BUGS section.

File: abrute [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                by a double dash: -- target.aes
+.PP
+-h, --help      Prints help information.
+.PP
+-v, --version   Prints version information.
+.SH EXAMPLES
+Decrypts a file with a range of 4, characters of 1234, and a target path to the
+example.file.aes
+
+abrute 4 1234 -- test/data/example.file.aes 1>/dev/null
```
Comment (danielpclark) [inline abrute]: You can remove ` 1>/dev/null` off of each example

File: abrute [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Decrypts a file with a range of 4, characters of 1234, an option of beginning
+with a character sequence of 2222, and a target path to the example.file.aes
+
+abrute 4 1234 -s 2222 -- test/data/example.file.aes 1>/dev/null
```
Comment (danielpclark) [inline abrute]: You can remove ` 1>/dev/null` off of each example

File: abrute [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+Decrypts a file with a range of 4, characters of 1234, an option of setting the
+allowed adjacent characters to be 0, and a target path ot the example.file.aes
+
+abrute 4 1234 -a 0 -- test/data/example.file.aes 1>/dev/null
```
Comment (danielpclark) [inline abrute]: You can remove ` 1>/dev/null` off of each example

File: abrute (hunk 3 above)
Comment (danielpclark) [reply abrute]: Also you can remove `test/data/`

File: abrute (hunk 4 above)
Comment (danielpclark) [reply abrute]: Also you can remove `test/data/`

File: abrute (hunk 5 above)
Comment (danielpclark) [reply abrute]: Also you can remove `test/data/`

**Review by danielpclark — CHANGES_REQUESTED** [review]
Looks good.  After addressing these changes please rename the file from `abrute` to `abrute.1` :+1:

File: abrute (hunk 1 above)
Comment (Alcamech) [reply abrute]: done

File: abrute (hunk 3 above)
Comment (Alcamech) [reply abrute]: done

File: abrute (hunk 2 above)
Comment (Alcamech) [reply abrute]: done

**Pull request merged**